apis/kueue/v1beta1/constants.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const (
+	ResourceInUseFinalizerName = "kueue.k8s.io/resource-in-use"
```

Comment:
Can we use only one domain type (`kueue.k8s.io` vs `kueue.x-k8s.io`)?

Reply:
oops, yes, this was a mistake in alpha.